src/main/zapHomeFiles/hudtutorial/en_GB/Index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Index</title>
+	<link href="tutorial.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+<div class="roundContainer">
+	<h1>Index</H1>
+
+	<!-- INDEX -->
+	
+	<p>
+	You can only go directly to pages if you have completed all of the tasks for the previous pages.
+	<p>
+	If you find this feature too annoying then you can disable it via the ZAP HUD Options page,
+	you will then be able to access all of the pages without completing the tasks. 
```

Comment:
This does not seem to be working correctly, even after selecting the `Skip tutorial tasks` the links are not shown for following pages.

Reply:
Doh! Fixed.